sonaranalyzer-dotnet/src/Tests/SonarAnalyzer.UnitTest/Metrics/ExecutableLinesCalculatorTest.cs

```diff
@@ -0,0 +1,251 @@
+﻿/*
+ * SonarAnalyzer for .NET
+ * Copyright (C) 2015-2017 SonarSource SA
+ * mailto: contact AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+
+using FluentAssertions;
+using Microsoft.CodeAnalysis.CSharp;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+namespace SonarAnalyzer.UnitTest.Common
+{
+    [TestClass]
+    public class ExecutableLinesCalculatorTest
+    {
+        private static void AssertLinesOfCode(int expectedLineCount, string code)

```

Comment:
Let's invert parameters order and call it with `expectedLineCount: 0` instead of only `0`.

Reply:
It is pretty clear from the name and both arguments what the method does. Also, having expected as a first parameter is a common convention.

It's not fluent, but to me that looks better than your suggestion.